packages/expo/ios/ExpoLiquidGlassConstantsModule.swift

```diff
@@ -0,0 +1,26 @@
+import ExpoModulesCore
+
+public class ExpoLiquidGlassConstantsModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoLiquidGlassConstants")
+
+    Constants([
+      "isLiquidGlassAvailable": ExpoLiquidGlassConstantsModule.isLiquidGlassAvailable()
+    ])
```

Comment:
The `Constants` component was deprecated, and it should be replaced with `Constant` or `Property` - see https://github.com/expo/expo/pull/38926

Reply:
Should we update the docs for modules API https://docs.expo.dev/modules/module-api/?